lib/src/widgets/chat_groupedlist_widget.dart

```diff
@@ -210,23 +205,19 @@ class _ChatGroupedListWidgetState extends State<ChatGroupedListWidget>
                   )
                 : _chatStreamBuilder,
           ),
-          widget.showTypingIndicator
-              ? TypingIndicator(
-                  typeIndicatorConfig: widget.typeIndicatorConfig,
-                  chatBubbleConfig: chatBubbleConfig?.inComingChatBubbleConfig,
-                  showIndicator: widget.showTypingIndicator,
-                )
-              : ValueListenableBuilder(
-                  valueListenable: ChatViewInheritedWidget.of(context)!
-                      .chatController
-                      .typingIndicatorNotifier,
-                  builder: (context, value, child) => TypingIndicator(
+          ValueListenableBuilder(
+            valueListenable: ChatViewInheritedWidget.of(context)!
+                .chatController
+                .typingIndicatorNotifier,
```

Comment:
Can you please just insert this widget if chatController is not null?
```suggestion
       if(chatController != null)   ValueListenableBuilder(
            valueListenable: chatController!.typingIndicatorNotifier,
```

Reply:
Well this is the same thing you do further below in the file. Should I fix this also?

Reply:
I fixed it.